assets/javascripts/kitten/components/steppers/vertical-progress/components/step.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { pxToRem } from '../../../../helpers/utils/typography'
+import { ScreenConfig } from '../../../../constants/screen-config'
+import { Status } from './status'
+
+const StyledItem = styled.li`
+  margin: ${pxToRem(15)} 0 ${pxToRem(25)} 0;
+
+  &:first-of-type {
+    margin-top: 0;
+    padding-top: 0;
+  }
+  &:last-of-type {
+    margin-bottom: 0;
+    padding-bottom: 0;
+  }
+
+  li:not(:last-of-type) {
+    margin: 0 !important;
+
+    &::after {
+      display: none;
+    }
+  }
+
+  .k-VerticalProgress--itemContainer {
+    display: inline-flex;
+    flex-wrap: wrap;
+    padding-right: ${pxToRem(10)};
+  }
+
+  .k-VerticalProgress--step__content {
+    padding-left:  ${pxToRem(50)};
+
+    @media (min-width: ${ScreenConfig.S.min}px) {
+      padding-left: ${pxToRem(82)};
+    }
+  }
+`
+
+export const Step = ({
+  actions,
+  waiting,
+  publish,
+  valid,
+  disabled,
+  statusProps,
+  progressTitle,
+  children,
+  ...other
+}) => {
+  return (
+    <StyledItem className="k-VerticalProgress--item">
+      <div className="k-VerticalProgress--itemContainer" {...other}>
```

Comment:
Par souci de respect de la syntaxe BEM, même chose plus bas avec `k-VerticalProgress--step__content`

```suggestion
    <StyledItem className="k-VerticalProgress__item">
      <div className="k-VerticalProgress__itemContainer" {...other}>
```

Reply:
En plus de ça, j'ai l'impression qu'on n'a pas besoin d'un `div` dans le `li`. Je vais regarder si ça ne pourrait pas alléger un peu la structure.